.github/workflows/choreo.yml

```diff
@@ -82,23 +82,23 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: ${{ matrix.artifact-name }}
-          path: target/**/release/bundle/nsis/*.exe
+          path: "**/*.exe"
```

Comment:
Are the quotes needed?

Reply:
Not sure, but my editor's syntax highlighting suggested that it needed quotes.